.env

```diff
@@ -31,3 +31,6 @@ PRICES_SERVICE_URI=prices.mainnet.teritori.com:443
 DISPLAYED_NETWORKS_IDS=teritori,teritori-testnet,ethereum,ethereum-goerli,cosmos-hub,juno,osmosis,osmosis-testnet,solana,gno-testnet
 SELECTABLE_NETWORKS_IDS=teritori,teritori-testnet,ethereum,ethereum-goerli,osmosis,osmosis-testnet,gno-testnet
 IMG_PROXY_URL=https://imgproxy.tools.teritori.com/insecure/
+
+AIRTABLE_API_KEY_DAPPS_STORE=patZYqro8XEjo8qXT.97183a7a2a21decc37eeafe8da99f669876df0df109bc908c6311f4302c1991a
```

Comment:
Is this safe to put this key here ? it's only for testing ?

Reply:
🙈